src/main/java/online/carsharing/repository/rental/RentalRepository.java

```diff
@@ -12,4 +14,8 @@ public interface RentalRepository extends JpaRepository<Rental, Long> {
 
     @Query("SELECT r FROM Rental r JOIN FETCH r.user JOIN FETCH r.car WHERE r.active = :isActive")
     List<Rental> findAllByActive(boolean isActive);
+
+    @Query("SELECT r FROM Rental r LEFT JOIN FETCH r.car LEFT JOIN FETCH"
+            + " r.user WHERE r.returnDate <= :tomorrow AND r.actualReturnDate IS NULL")
+    List<Rental> findOverdueRentals(@Param("tomorrow") LocalDate tomorrow);
```

Comment:
It is not necessary to write `@Param` before the `tomorrow` parameter. I tested this method it works fine.
```suggestion

    @Query("SELECT r FROM Rental r LEFT JOIN FETCH r.car LEFT JOIN FETCH"
            + " r.user WHERE r.returnDate <= :tomorrow AND r.actualReturnDate IS NULL")
    List<Rental> findOverdueRentals(LocalDate tomorrow);
```